resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -10,9 +10,12 @@
     executor:
       docker:
         image: resolwebio/rnaseq:3.7.0
-  data_name: "Novel Junctions ({{ star|sample_name|default('') }}{{ extracted|sample_name|default('') }})"
-  version: 0.0.1
-  type: data:bed:junctions
+  data_name: >
+    "Novel Junctions ({{ alignment_star|sample_name|default('?') }}
```

Comment:
We don't write the analysis name in the data name anymore, because the analysis is always displayed in the column next to the name. I suggest imply
```
{{ alignment_star|sample_name|default('?') }}  
{{ alignment|sample_name|default('?') }}
{{ input_bed_junctions.bed.file|default('?') }})"
```

Reply:
How will this render? I guess this way:
  - if `alignment_star` is given, text will be displayed as `"sample??"`
  - if `alignment` is given, text will be displayed as `"?sample?"`
  - if `input_bed_junctions` is given, text will be displayed as `"??bed_file"`

Consider replacing `'?'` with `''`.